nimbus/examples/experiment.rs

```diff
@@ -324,12 +325,15 @@ fn main() -> Result<()> {
                 // options.
                 let uuid = uuid::Uuid::new_v4();
                 let aru = AvailableRandomizationUnits::with_client_id(&client_id);
-                let enrollment =
-                    nimbus::evaluate_enrollment(&uuid, &aru, &Default::default(), &exp)?;
-                results.insert(
-                    enrollment.status.clone(),
-                    results.get(&enrollment.status).unwrap_or(&0) + 1,
-                );
+                let enrollment = nimbus::evaluate_enrollment(&uuid, &aru, &context.clone(), &exp)?;
+                let key = match enrollment.status.clone() {
```

Comment:
I'd have thought the output before this part of the patch would be better than after, but 🤷

Reply:
Because of the addition of the enrollment_id, it was basically categorizing each enrollment by itself with a count of one, so I just simplified what it was categorizing by again.

Reply:
It made it difficult to see how many enrolled and didn't enroll